lib/model/narrow.dart

```diff
@@ -105,9 +106,10 @@ class TopicNarrow extends Narrow implements SendableNarrow {
   TopicNarrow sansWith() => TopicNarrow(streamId, topic);
 
   @override
-  bool containsMessage(Message message) {
-    return (message is StreamMessage
-      && message.streamId == streamId && message.topic == topic);
+  bool containsMessage(MessageBase message) {
+    final conversation = message.conversation;
+    return conversation is StreamConversation
```

Comment:
And similarly I guess these other three `containsMessage` implementations should gain a test case each to verify that they *do* include pseudo-messages when the conversation matches. Or more precisely two cases each: one positive, one negative, to show that the result depends on the conversation.